# HG changeset patch
# User Rik <rik@octave.org>
# Date 1385049785 28800
#      Thu Nov 21 08:03:05 2013 -0800
# Node ID c1e4e25872f7d151f550867da33a5bdc39a09eda
# Parent  43c199e83ed7a0e90f29dd3a03b3274cacd84022
syscalls.cc: Pipe %! system() call results to NUL on Windows platforms.

* syscalls.cc: Pipe %! system() call results to NUL on Windows platforms.

diff --git a/libinterp/corefcn/syscalls.cc b/libinterp/corefcn/syscalls.cc
--- a/libinterp/corefcn/syscalls.cc
+++ b/libinterp/corefcn/syscalls.cc
@@ -377,17 +377,17 @@ exit status, it will linger until Octave
   return retval;
 }
 
 /*
 %!test
 %! unix_sort = true;
 %! cmd = {"sort", "-r"};
 %! if (ispc ())
-%!   status = system ("sort /? 2> NUL");
+%!   status = system ("sort /? 2>NUL 1>NUL");
 %!   if (status == 0)
 %!     unix_sort = false;
 %!     cmd = {"sort", "/R"};
 %!   endif
 %! endif
 %! [in, out, pid] = popen2 (cmd{:});
 %! if (isunix ())
 %!   EAGAIN = errno ("EAGAIN");
